plugins/out_azure/azure.c

```diff
@@ -319,13 +320,35 @@ static int cb_azure_exit(void *data, struct flb_config *config)
     return 0;
 }
 
+/* Configuration properties map */	
+static struct flb_config_map config_map[] = {	
+    {	
+     FLB_CONFIG_MAP_STR, "customer_id", NULL,	
+     0, FLB_TRUE, offsetof(struct flb_azure, customer_id),	
+     NULL	
+    },	
+    {	
+     FLB_CONFIG_MAP_STR, "shared_key", NULL,	
+     0, FLB_TRUE, offsetof(struct flb_azure, shared_key),	
+     NULL	
+    },
+    {	
+     FLB_CONFIG_MAP_STR, "log_type", "fluentbit",	
+     0, FLB_TRUE, offsetof(struct flb_azure, log_type),	
+     NULL	
+    },
+    
```

Comment:
See line 102 of azure_conf.c, there is another option named `time_key`
available for out_azure. So you need to `time_key` option to the list.

Reply:
(Or it would become unusable. Try `fluent-bit -i dummy -o azure -p time_key=time` for
 confirmation)

Reply:
(Or it would become unusable. Try running `fluent-bit -i dummy -o azure -p time_key=time` for confirmation)

Reply:
We would need to update the documentation here then https://docs.fluentbit.io/manual/output/azure It is not mentioned here.

Reply:
> We would need to update the documentation here then

That's true. Can you submit a patch to https://github.com/fluent/fluent-bit-docs?
This will make a good contribution.

Reply:
Done : https://github.com/fluent/fluent-bit-docs/pull/283

Reply:
Thank you. Looks good to me.